cql3/Cql.g

```diff
@@ -2387,6 +2410,10 @@ K_MUTATION_FRAGMENTS:    M U T A T I O N '_' F R A G M E N T S;
 
 K_VECTOR_SEARCH_INDEXING: V E C T O R '_' S E A R C H '_' I N D E X I N G;
 
+K_SIMILARITY_EUCLIDEAN:     S I M I L A R I T Y '_' E U C L I D E A N;
```

Comment:
I think you need to add these new keywords to unreserved_keyword? If not, why not?

Reply:
Ok. I'll resolve the discussion.